charts/tidb-cluster/values.yaml

```diff
@@ -473,6 +473,7 @@ monitor:
     imagePullPolicy: IfNotPresent
     service:
       type: NodePort
+      portName: reloader
```

Comment:
Shall we make the default on follow the requirements of istio?

Reply:
update.